inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -456,122 +430,130 @@ population_data_age_months <- gen_population(total_pop = 5000,
+linelist.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+```{r lab_data}
+## Read data -------------------------------------------------------------------
+## This step reads in your lab data from Excel.
+## You may need to rename your columns.
+
 ## to read in a specific sheet use "which"
 # lab_data <- rio::import(here("labresults.xlsx"), which = "Sheet1")
 
-## repeat same cleaning steps as in standardise_clean_data code chunk as appropriate
+## repeat preparation steps as appropriate
 ## make sure your unique ID variable name matches!
 
-## Fake data ------------------------------------ 
-## THE BELOW CAN BE DELETED IF YOU HAVE YOUR OWN (REAL) LABORATORY DATA! 
-## its just to be able to demonstrate script in this template
-
-# # generate artificial lab tests results
-# lab_results <- linelist_cleaned %>% 
-#                 select(case_number) %>% 
-#                 mutate(test_result = sample(c("Positive", "Negative"),
-#                                             nrow(linelist_cleaned), 
-#                                             replace = TRUE)
-#                        )
-# 
-# # merging linelist with lab dataset 
+## Merge lab to linelist -------------------------------------------------------
 # linelist_cleaned <- left_join(linelist_cleaned, lab_results, 
 #                               by = "case_number")
 ```
 
 
 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+browse_data
 
+You'll want to look at your data. Here are a few ways you can explore.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 ```{r browse_data, eval = FALSE}
-## Browsing data ---------------------------------
-## here are a few ways to do data explorations 
-
 ## view the first ten rows of data
 head(linelist_cleaned, n = 10)
 
-## view your whole dataset interactivley (in an excel style format)
-## Remember that `View` needs to be written with a capital *V*
+## view your whole dataset interactively (in an excel style format)
 View(linelist_cleaned)
 
 ## overview of variable types and contents
 str(linelist_cleaned)
 
-## gives mean, median and max values of variables
-## gives counts for categorical variables
+## get summary: 
+## mean, median and max values of numeric variables
+## counts for categorical variables
 ## also gives number of NAs
 summary(linelist_cleaned)
 
 ## view unique values contained in variables 
-unique(linelist_cleaned$sex)
+## you can run this for any column -- just replace the column name
+unique(linelist_cleaned$patient_origin) 
 
-## another alternative is with the "summarytools package"
-## use the dfSummary function in combination with view
-## note that view is not capitalised with this package
-# install.packages("summarytools")
-# summarytools::view(summarytools::dfSummary(linelist_cleaned))
 ```
 
 
 
 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+remove_unused
```

Comment:
maybe remove_unused_data - thought at first this was a tag to delete the section.